circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/CopierFactoryManager.java

```diff
@@ -55,13 +56,21 @@ void postConstruct() {
     }
   }
 
-  CopierFactory getCopierFactory(Path sourceLocation, Path replicaLocation) {
+  CopierFactory getCopierFactory(Path sourceLocation, Path replicaLocation, Map<String, Object> copierOptions) {
     String sourceScheme = sourceLocation.toUri().getScheme();
     String replicaScheme = replicaLocation.toUri().getScheme();
     for (CopierFactory copierFactory : copierFactories) {
-      if (copierFactory.supportsSchemes(sourceScheme, replicaScheme)) {
+      final String copierFactoryName = copierFactory.getClass().getName();
+      if (copierOptions.containsKey("copier-name")) {
```

Comment:
Please also update the README.md to document this new configuration option.

Reply:
Can you also convert this literal to a public constant? You can leave it in this class